tests/spectron/spectronSetup.js

```diff
@@ -30,18 +31,33 @@ class App {
         this.app = new Application(this.options);
     }
 
-    startApplication(configurations) {
-        return this.app.start().then((app) => {
-            if (configurations)
-            {
-                if (configurations.alwaysOnTop)  {
-                    app.browserWindow.setAlwaysOnTop(true);
+    async startApplication(configurations) {
+        try {
+            this.app = await this.app.start();
+            await this.app.client.waitForVisible(ui.TITLE_BAR, 60000);
```

Comment:
@thaisym1912 This will not work on `Mac OS` and `Windows 7` as they don't have a custom title bar

Reply:
Updated